changelog-maker.js

```diff
@@ -1,13 +1,14 @@
 #!/usr/bin/env node
 
-const gitcmd        = 'git log --pretty=full --since="{{sincecmd}}" --until="{{untilcmd}}"'
+var gitcmd        = 'git log --pretty=full --since="{{sincecmd}}" --until="{{untilcmd}}"'
```

Comment:
please line up the `=` here and elsewhere you've made changes, also as I mentioned `const` should not be a problem (I've been using it for ages)